haystack/components/embedders/sentence_transformers_text_embedder.py

```diff
@@ -47,26 +49,28 @@ def __init__(
         Create a SentenceTransformersTextEmbedder component.
 
         :param model:
-            Local path or ID of the model on HuggingFace Hub.
+            The model to use for calculating embeddings.
+            Specify the path to a local model or the ID of the model on Hugging Face.
         :param device:
             Overrides the default device used to load the model.
         :param token:
-            The API token used to download private models from Hugging Face.
+            To use private models from Hugging Face, enter an API token here.
```

Comment:
```suggestion
            An API token to use private models from Hugging Face.
```

Feels like a more explicit parameter description.